### PR TITLE
Implemented ReadonlyTransactionManager sendCall

### PR DESCRIPTION
**Pull request by AlexandrouR**
### What does this PR do?
Implemented sendCall method in the ReadonlyTransactionManager

### Where should the reviewer start?
ReadonlyTransactionManager class , sendCall method.

### Why is it needed?
Addresses issue #933

### CODE REVIEW

File: core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ReadonlyTransactionManagerTest {
+
+    EthCall response = mock(EthCall.class);
+    Request request = mock(Request.class);
+    Web3j web3j = mock(Web3j.class);
```
Comment (josh-richardson) [inline core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java]: It's probably worth mentioning that everything here is mocked, so as far as I can tell nothing is actually being tested. That said this stuff is hard to test in an atomic way.

Comment (iikirilov) [reply core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java]: Agreed. I think the approach should mock the `Web3jService` instance passed to `Web3j.build(...)`

Comment (iikirilov) [reply core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java]: Something like:

```
Web3jService service = mock(Web3jService.class);
when(service.send()).thenReturn("test");
Web3j web3j = web3j.build(service);
```

Then you can create the ReadonlyTransactionManager with a concrete instance of web3j.

File: core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+public class ReadonlyTransactionManagerTest {
+
+    Web3jService service = mock(Web3jService.class);
+    Web3j web3j = Web3j.build(service);
+    Transaction transaction = mock(Transaction.class);
+    DefaultBlockParameter defaultBlockParameter = mock(DefaultBlockParameter.class);
+    EthCall response = mock(EthCall.class);
+
+    Request<?, EthCall> request =
```
Comment (iikirilov) [inline core/src/test/java/org/web3j/tx/ReadonlyTransactionManagerTest.java]: Is this required? I don't see it being used in the test.

**Pull request merged**